evap/rewards/views.py

```diff
@@ -41,12 +41,20 @@ def redeem_reward_points(request):
     try:
         save_redemptions(request, redemptions, previous_redeemed_points)
         messages.success(request, _("You successfully redeemed your points."))
-    except (NoPointsSelected, NotEnoughPoints, RedemptionEventExpired) as error:
-        messages.warning(request, error)
-        return 400
-    except OutdatedRedemptionData as error:
-        messages.error(request, error)
-        return 409
+    except (NoPointsSelected, NotEnoughPoints, RedemptionEventExpired, OutdatedRedemptionData) as error:
+        error_string = ""
+        status_code = 400
+        if isinstance(error, NoPointsSelected):
+            error_string = "You cannot redeem 0 points."
```

Comment:
Codecov indicates that we have no test for this case. Could you add one? It should be fairly easy, check out `evap/rewards/tests/test_views.py` around line 63 for the tests of the other cases.